lib/Main.js

```diff
@@ -191,16 +191,21 @@ Main.prototype.getBotIntent = function() {
     return this._bridge.getIntent();
 };
 
+Main.prototype.getUserId = function(id, team_domain) {
+    return [
+        "@", this._config.username_prefix, team_domain.toLowerCase(),
+            "_", id.toUpperCase(), ":", this._config.homeserver.server_name
```

Comment:
should start considering making this lowercase, although that's a PR for a later time.